### PR TITLE
Add support for Qt5 + python bindings for GraphViz and QCustomPlot

### PR DESCRIPTION
**Pull request by jmirabel**
#### Note about Qt support:
- the GUI support at least Qt 4.8, 5.2 and 5.5,
- with Qt version between 5 and 5.4, there is not support for Python.

#### Note about GraphViz:
- it depends upon:
  - https://github.com/humanoid-path-planner/pythonqt, branch `qt4` or `qt5`.
  - https://github.com/humanoid-path-planner/qgv

#### Note about QCustomPlot:
- the license is GPL,
- it is disabled by default,
- the bindings are stable but incomplete.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -16,7 +16,7 @@
 # <http://www.gnu.org/licenses/>.
 
 # Requires at least CMake 2.6 to configure the package.
```
Comment (jcarpent) [inline CMakeLists.txt]: 2.8

**Comment by jcarpent** [comment]
Use QT4 by default.

**Comment by jmirabel** [comment]
The example can be launched with `gepetto-gui -p libpyqcustomplot.so -q examples/pyplugin/qcustomplot_example.py`

**Comment by jmirabel** [comment]
The huge amount of lines added correspond mainly to the introduction of QCustomPlot (2 files, +28,886 lines, commit 60dbad5 ).

On Ubuntu 14.04, there is a very old version of QCustomPlot. Adding this 2 files was the simplest way I found. I am open to any better method.

**Comment by jcarpent** [comment]
Works fine on Mac OS. I will merge.

**Pull request merged**